release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -30,9 +30,364 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### Support for the spread operator in the list constructor
+
+Introduced the spread operator support for the list constructor expression.
+
+If the spread operator in a list constructor expression is `...x`, then, `x` is expected to be a list (i.e., an array or a tuple). All the member values of the list that result from evaluating `x` are included in the list value being constructed.
+
+```ballerina
+import ballerina/io;
+
+public function fn() {
+    int[] a1 = [3, 4];
+    int[] v1 = [1, 2, ... a1];
+    io:println(v1); // [1, 2, 3, 4]
+
+    int[2] a2 = [6, 7];
+    int[] v2 = [1, 2, ... a1, 5, ... a2];
+    io:println(v2); // [1, 2, 3, 4, 5, 6, 7]
+
+    [int, string] t1 = [5, "s"];
+    any[] v3 = [ ... t1, "x"];
+    io:println(v3); // [5, "s", "x"]
+
+    [boolean, int...] t2 = [false, 4, 7];
+    [string, int, string, boolean, int...] v4 = ["x", ... t1, ... t2];
+    io:println(v4); // ["x", 5, "s", false, 4, 7];
+
+    var v5 = [4, ... t1, ... a2];
+    io:println(v5); // [4, 5, "s", 6, 7];
+}
+```
+
+The spread operator is not allowed with a varying-length list if the inherent type of the list being constructed has required members that are not guaranteed to have been provided a value.
+
+```ballerina
+public function fn() {
+    [int, string...] t1 = [5, "s"];
+    [int, string, string...] v1 = [ ... t1]; // results in an error since a value is not guaranteed to have been provided for the second tuple member
+
+    [int, boolean, string, int...] t2 = [5, false, "w"];
+    [int, boolean, anydata...] v2 = [ ... t2, "x", "y"]; // works as all fixed tuple members are guaranteed to have been provided values
+}
+```
+
+##### Allow `int*float`, `float*int`,` int* decimal`, `decimal*int`, `float/int`, `decimal/int`, `float%int`, and `decimal%int`.
+
+Multiplicative expression is now allowed with `int`, `float` and `int`, and `decimal` operands.  The resulting type of the expression will be the fractional type.
+
+This allows the below.
+- Multiplication supports both `int*float` and ``float*int` similar to `decimal*float` and `decimal*int`.
```

Comment:
```suggestion
- Multiplication supports both `int*float` and `float*int` similar to `decimal*float` and `decimal*int`
```
We don't put full stops in bullet points. This is just a best practice to assure consistency.